src/View/Helper.php

```diff
@@ -219,4 +219,23 @@ public function implementedEvents()
         }
         return $events;
     }
+
+    /**
+     * Returns an array that can be used to describe the internal state of this
+     * object.
+     *
+     * @return array
+     */
+    public function __debugInfo()
+    {
+        return [
+            'helpers' => $this->helpers,
+            'theme' => $this->theme,
+            'plugin' => $this->plugin,
+            'fieldset' => $this->fieldset,
+            'tags' => $this->tags,
```

Comment:
I think we forgot to remove these properties :cry: They aren't actually used for anything.